provider/resources.go

```diff
@@ -115,6 +115,10 @@ func Provider() tfbridge.ProviderInfo {
 					"size": {
 						Type: makeType(digitalOceanMod, "DatabaseSlug"),
 					},
+					"urn": {
+						Name:       "doUrn",
+						CSharpName: "DoUrn",
```

Comment:
I think it would read better if these incorporated the name of the type they're associated with.

```suggestion
						Name:       "clusterUrn",
						CSharpName: "ClusterUrn",
```

Also, is the `CSharpName` actually necessary? Shouldn't the code generator pick up the custom name and `PascalCase` it?

Reply:
I agree. The "do" prefix (for Digital Ocean) can read as a verb which sounds strange.

Reply:
Will update them now :) Wanted to put something out there as a talking point